src/components/MovieList/index.ts

```diff
@@ -26,12 +26,19 @@ class MovieList {
 
   #$skeletonContainer = getSkeletonContainer();
 
-  constructor($target: Element) {
+  constructor($target: Element, onClickMovie: (movieId: number) => void) {
     this.#$target = $target;
     this.renderSkeleton();
     this.init();
-
     $(".btn").addEventListener("click", this.onClickMoreButton.bind(this));
+    $(".item-list").addEventListener("click", (event) => {
+      const closestLi = (event.target as HTMLElement).closest("li");
+
+      if (closestLi?.dataset.id) {
+        const movieId = parseInt(closestLi.dataset.id, 10);
```

Comment:
```suggestion
        const movieId = Number(closestLi.dataset.id);
```

(소소) parseInt 와 Number 의 차이는 무엇일까요?

Reply:
parseInt는 문자열 인자를 파싱해서 특정 진수(위 코드에서는 10진수)의 정수를 반환합니다.
Number는 문자열에서 숫자의 형태를 가지고 있으면 음수 및 소수점까지 반환하고 문자가 들어있으면 NaN을 반환합니다.

Reply:
👍🏻 👍🏻 

논외) `parseInt(0.000000000000005)` 은 뭘까요?